commands/init.go

```diff
@@ -115,15 +119,25 @@ func (a *initFnCmd) init(c *cli.Context) error {
 		dir = a.wd
 	}
 
+	if a.triggerType != "" {
```

Comment:
is this using the trigger type to detect version? Shouldn't we only be init'ing fns of the new version?